src/Components/index.js

```diff
@@ -2,6 +2,7 @@ import Nav from "./Nav";
 import Hero from "./Hero";
 import Auth from "./UserAuthPage"
 import SignUp from "./SignUp"
+import UploadFiles from "./UploadFiles"
 
 
-export { Nav, Hero, Auth, SignUp, }
\ No newline at end of file
+export { Nav, Hero, Auth, SignUp, UploadFiles,  }
```

Comment:
Is it standard to add the last comma? If not, could we keep it clean by not including it in code that is pushed?

Reply:
It's called a trailing comma, and it just helps version control as changes can be made and the lac thereof could break the app. I would suggest leave it to prevent any future mishaps

Reply:
Sounds good, then lets leave it in. Thanks for the clarification